build/generate_bundle.sh

```diff
@@ -48,8 +48,8 @@ build_bundle_instructions() {
 # generate templates
 echo "## Begin bundle creation"
 generate_version
-create_working_dir
-generate_dockerfile
+#create_working_dir
```

Comment:
I need to continue to do some clean up and make sure I remove items that are commented out and no longer in use.